src/components/client-add/ClientAdd.jsx

```diff
@@ -30,18 +30,18 @@ const ClientAdd = ({ addFormData, handleFormVisible }) => {
   };
 
   const handleCloseForm = () => {
-    handleFormVisible();
     setBranch('');
+    setBranchAdress('');
     setClient('');
     setLocality('');
-    setBranchAdress('');
     setPostalCode('');
+    toggleFormVisibility();
   };
 
   const handleSubmit = event => {
     event.preventDefault();
 
-    addFormData({
+    addClientData({
```

Comment:
Don't use abstract names like data, even not when preceded by a word like "client", it's simply "add client".